backend-go/internal/jwt/payload.go

```
@@ -1,8 +1,7 @@
 package jwt
 
 import (
-	"net/http"
-
+	"errors"
```

golangci-lint (1.62.2)

File is not `goimports`-ed with -local github.com/yorkie-team/codepair

(goimports)